ros2topic/ros2topic/verb/echo.py

```diff
@@ -97,8 +99,10 @@ def represent_ordereddict(dumper, data):
 def subscriber(node, topic_name, message_type, callback):
     if message_type is None:
         topic_names_and_types = get_topic_names_and_types(node=node)
+        expanded_name = expand_topic_name(topic_name, node.get_name(), node.get_namespace())
+        validate_full_topic_name(expanded_name)
```

Comment:
This function might raise an `InvalidTopicNameException` which I assume results in printing a stacktrace which is not a good user experience. The possible exceptions should be catched and a "nicer" error message (without a stacktrace) should be printed instead.

Reply:
Yeah, when I made that suggestion I meant you could raise it here, catch it and do something nicer. If you just want the traceback, then that will happen when you try to create a pub or sub with the topic.

Reply:
Ah, OK.  I'll open a new PR with something nicer.

Reply:
#58